ios/Exponent/Versioned/Core/EXVersionManager.mm

```diff
@@ -273,7 +273,7 @@ - (void)configureABIWithFatalHandler:(void (^)(NSError *))fatalHandler
                          logFunction:(RCTLogFunction)logFunction
                         logThreshold:(NSInteger)threshold
 {
-  RCTEnableTurboModule(YES);
+  RCTEnableTurboModule([self.params[@"manifest"][@"ios"][@"enableExperimentalTurboModules"] boolValue]);
```

Comment:
Change to `experiments.turboModules` (we have `experiments.redesignedLogBox` already): https://docs.expo.io/versions/latest/config/app/#experiments